src/baseWrapper.ts

```diff
@@ -41,7 +41,7 @@ export default abstract class BaseWrapper<ElementType extends Node>
     return this.wrapperElement
   }
 
-  constructor(element: ElementType) {
+  protected constructor(element: ElementType) {
```

Comment:
I am not sure this does anything - this class is already `abstract` so you cannot make an instance of it anyway. Might be good to revert an unnecessary change, unless I'm missing something?